packages/medusa/src/api/admin/inventory-items/validators.ts

```diff
@@ -72,7 +72,6 @@ export const AdminCreateInventoryLocationLevel = z
     location_id: z.string(),
     stocked_quantity: z.number().min(0).optional(),
     incoming_quantity: z.number().min(0).optional(),
-    reserved_quantity: z.number().min(0).optional(),
```

Comment:
@riqwan the tests used to create this as well, was that a mistake?

Reply:
Yeah, must've been created at a time when reservations weren't available yet.